app/crate/worker_handlers/acquisition.py

```
@@ -1123,10 +1139,101 @@ def _handle_library_upload(task_id: str, params: dict, config: dict) -> dict:
+    emit_task_event(task_id, "info", {
+        "message": (
+            f"M4A fix complete: {deleted} intermediates deleted ({mb_freed:.0f} MB freed), "
+            f"{converted} remuxed to FLAC, {failed} failed"
+        ),
+    })
+
+    if (deleted > 0 or converted > 0) and not dry_run:
+        from crate.library_sync import start_scan
+        try:
+            start_scan()
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)